crates/astria-core/src/execution/v1alpha2/mod.rs

```diff
@@ -2,9 +2,107 @@ use prost_types::Timestamp;
 
 use crate::{
     generated::execution::v1alpha2 as raw,
+    sequencer::v1alpha1::{
+        IncorrectRollupIdLength,
+        RollupId,
```

Comment:
side remark: we should probably move these out of sequencer into a `common` module